python/tests/operations/abs.py

```diff
@@ -0,0 +1,218 @@
+import os
+
+import numpy as np
+
+import equistore
+import equistore.io
+from equistore import Labels, TensorBlock, TensorMap
+
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
+TEST_FILE = "qm7-spherical-expansion.npz"
+
+
+class TestAbs:
```

Comment:
```suggestion
class TestAbs:
    """
    Unit tests for the :py:func:`equistore.abs` function.
    
    Note that unit tests for values/gradients containing complex 
    numbers is currently untested.
    """
```

Reply:
(or perhaps just write a simple unit test for a TensorMap with complex numbers)

Reply:
?